src/object-attribute.scss

```
@@ -0,0 +1,43 @@
+@import "./new-settings";
+@import "./mixins";
+
+$block: #{$fd-namespace}-object-attribute;
+
+.#{$block} {
+  @include fd-reset();
+  @include fd-ellipsis();
+
+  color: var(--sapContent_LabelColor);
+  display: block;
+
+  &--link {
+    @include action-cursor();
+
+    text-decoration: none;
+    color: var(--sapLinkColor);
+
+    &:visited {
+      color: var(--sapLink_Visited_Color);
+    }
+
+    &:hover {
+      @include fd-hover() {
+        text-decoration: underline;
+        color: var(--sapLink_Hover_Color);
+      }
+    }
+
+    &:focus {
+      @include fd-focus();
+      @include fd-fiori-focus($offset: -0.0625rem);
+    }
+
+    &[disabled],
+    &[aria-disabled="true"] {
+      @include fd-disabled() {
+        opacity: var(--sapContent_DisabledOpacity);
+        pointer-events: none;
+      }
+    }
```

You can omit these selectors, as they are already provided in the mixin.

```suggestion
    @include fd-hover() {
      text-decoration: underline;
      color: var(--sapLink_Hover_Color);
    }

    @include fd-focus();
    @include fd-fiori-focus($offset: -0.0625rem);

    @include fd-disabled() {
      opacity: var(--sapContent_DisabledOpacity);
      pointer-events: none;
    }
```